### PR TITLE
Issues/24 non conditional update

### LINKED ISSUE #24: Support non-conditional update of DocumentReferences and Transfered data
The data transfer process should be extended to support non-conditional update of DocumentReferences and transfered data like for example MeasureReport

This is necessary as not all FHIR servers support conditional updates.

Specifically, in order to transfer the MeasureReport from https://github.com/medizininformatik-initiative/fhir-data-evaluator/issues/41 via the data transfer process on a recurring basis requires the data transfer process to work with the Blaze FHIR server used by the FDPG.

### PR DESCRIPTION
**Pull request by wetret**
resolves #24

### CODE REVIEW

File: src/main/java/de/medizininformatik_initiative/process/data_transfer/service/InsertData.java [hunk 1]
```diff
@@ -116,6 +121,48 @@ private List<IdType> storeData(Variables variables, FhirClient fhirClient, Bundl
...
+				.filter(Bundle.BundleEntryComponent::hasRequest).filter(Bundle.BundleEntryComponent::hasResource)
+				.forEach(e ->
+				{
+					e.getRequest().setMethod(Bundle.HTTPVerb.PUT)
+							.setUrl(ResourceType.DocumentReference.name() + "/" + existingDocumentReferenceId);
+					e.getResource().setId(existingDocumentReferenceId);
+				});
+
+		return bundle;
+	}
```
Comment (hhund) [inline src/main/java/de/medizininformatik_initiative/process/data_transfer/service/InsertData.java:164]: Consider moving to `mii-processes-common` module.

**Review by hhund — APPROVED** [review]

**Review by juliangruendner — APPROVED** [review]
@wetret  - tested with blaze dic1 and dms in local setup, with two MeasureReports.
Worked as intended.

File: src/main/java/de/medizininformatik_initiative/process/data_transfer/service/InsertData.java (hunk 1 above)
Comment (wetret) [reply src/main/java/de/medizininformatik_initiative/process/data_transfer/service/InsertData.java:164]: Good idea, with implementation of the DSF API v2 for process development, we will also redesign the `mii-processes-common` repository. I will move this code as part of the redesign.

**Comment by wetret** [comment]
used wrong merge destination `main`. Branch is merged.

**Pull request closed without merging**